### DISCUSSION TITLE
Zigbee2mqtt Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open

### DISCUSSION THREAD

**Discussion by Balabonov73** [Koenkk/zigbee2mqtt#21748] [General]
Hello,

i can't launch Zigbee2mqtt on a RaspberryPi OS (fresh install)

The USB stick is recognised

ls -l /dev/serial/by-id/
total 0 lrwxrwxrwx 1 root root 13 Mar 10 15:12 usb-dresden_elektronik_ConBee_III_DE03188475-if00-port0 -> ../../ttyUSB0 

But i have this issue 

> Zigbee2MQTT:info 2024-03-10 15:16:08: Logging to console and directory: '/opt/zigbee2mqtt/data/log/2024-03-10.15-16-08' filename: log.txt
Zigbee2MQTT:info 2024-03-10 15:16:09: Starting Zigbee2MQTT version 1.36.0 (commit #2a53b8e5)
Zigbee2MQTT:info 2024-03-10 15:16:09: Starting zigbee-herdsman (0.35.1)
Zigbee2MQTT:error 2024-03-10 15:16:09: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2024-03-10 15:16:09: Failed to start zigbee
Zigbee2MQTT:error 2024-03-10 15:16:09: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2024-03-10 15:16:09: Exiting...
Zigbee2MQTT:error 2024-03-10 15:16:09: Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open usb-dresden_elektronik_ConBee_III_DE03188475-if00-port0'
at SerialPort. (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/serialPort.ts:28:28)
at SerialPort._error (/opt/zigbee2mqtt/node_modules/@serialport/stream/dist/index.js:82:22)
at /opt/zigbee2mqtt/node_modules/@serialport/stream/dist/index.js:118:18

Anyone can help me ?

BR

**Comment by francisp2** [comment]
Make sure neither ZHA or Deconz/Phoscon is installed.
